storage/txcache/score_test.go

```diff
@@ -79,7 +98,60 @@ func TestDefaultScoreComputer_scoreFluctuatesDeterministicallyWhileTxListForSend
 	scoreNone = int(computer.computeScore(list.getScoreParams()))
 
 	require.Equal(t, 0, scoreNone)
-	require.Equal(t, 17, scoreA)
-	require.Equal(t, 8, scoreAB)
-	require.Equal(t, 5, scoreABC)
+	require.Equal(t, 18, scoreA)
+	require.Equal(t, 12, scoreAB)
+	require.Equal(t, 10, scoreABC)
+}
+
+func TestDefaultScoreComputer_DifferentSenders(t *testing.T) {
+	txGasHandler, txFeeHelper := dummyParamsWithGasPrice(100 * oneBillion)
+	computer := newDefaultScoreComputer(txFeeHelper)
+
+	A := createTxWithParams([]byte("a"), "a", 1, 128, 50000, 100*oneBillion)    // min value normal tx
```

Comment:
Also, use a more real value (~ order of billions)?

Reply:
done